korman/properties/modifiers/gui.py

```diff
@@ -330,3 +333,302 @@ def _create_moul_nodes(self, clickable_object, nodes, journalnode, age_name):
+                                       poll=idprops.poll_mesh_objects)
+    clickable = PointerProperty(name="Clickable",
+                                description="The object the avatar will click on to activate the Linking Book GUI",
+                                type=bpy.types.Object,
+                                poll=idprops.poll_mesh_objects)
+
+    # -- Path of the Shell options --
+    # Popup Appearance
+    book_cover_image = PointerProperty(name="Book Cover",
+                                       description="Image to use for the Linking Book's cover",
+                                       type=bpy.types.Image,
+                                       options=set())
+    book_scale_w = IntProperty(name="Book Width Scale",
+                               description="Width scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    book_scale_h = IntProperty(name="Book Height Scale",
+                               description="Height scale",
+                               default=100, min=0, max=100,
+                               subtype="PERCENTAGE")
+    link_panel_image = PointerProperty(name="Linking Panel",
+                                       description="Image to use for the Linking Panel",
+                                       type=bpy.types.Image,
+                                       options=set())
+
+    stamped_book = BoolProperty(name="Has Stamp",
+                                description="Enables a stamp on the page opposite the book's linking panel",
+                                default=False)
+    stamp_image = PointerProperty(name="Stamp Image",
+                                  description="Image to use for the stamp",
+                                  type=bpy.types.Image,
+                                  options=set())
+    stamp_x = IntProperty(name="Stamp Position X",
+                          description="Width scale",
+                          default=140,
+                          subtype="UNSIGNED")
+    stamp_y = IntProperty(name="Stamp Position Y",
+                          description="Height scale",
+                          default=255,
+                          subtype="UNSIGNED")
+
+    def export(self, exporter, bo, so):
+        our_versions = (globals()[j] for j in self.versions)
+        version = exporter.mgr.getVer()
+        if version not in our_versions:
+            # We aren't needed here
+            exporter.report.port("Object '{}' has a LinkingBookMod not enabled for export to the selected engine.  Skipping.",
+                                 bo.name, version, indent=2)
+            return
+
+        if self.clickable is None:
+            raise ExportError("Linking Book modifier requires a clickable!", bo.name)
+
+        if self.seek_point is None:
+            raise ExportError("Linking Book modifier requires a seek point!", bo.name)
+
+        if version == pvPots:
```

Comment:
Even though this modifier doesn't support ABM, I would feel better if this were `if version <= pvPots:`